src/Cassandra/Data/Linq/CqlExpressionVisitor.cs

```diff
@@ -72,15 +86,19 @@ internal class CqlExpressionVisitor : ExpressionVisitor
         private int _limit;
         private readonly List<Tuple<PocoColumn, object, ExpressionType>> _projections = new List<Tuple<PocoColumn, object, ExpressionType>>();
         private readonly List<Tuple<string, bool>> _orderBy = new List<Tuple<string, bool>>();
-        private readonly List<string> _selectFields = new List<string>();
+        private readonly List<string> _selectFields = new List<string>(DefaultClauseParameterCapacity);
```

Comment:
It could be a good idea but we don't know how many of those fields are going to be selected until we evaluate the query.